README.Rmd

```diff
@@ -99,6 +97,26 @@ drive_chickwts
 ```
 Much better!
 
+#### Publish files
+
+Revisions of Google Documents, Sheets, and Presentations can be published online. By default, `drive_publish()` will publish your most recent revision. Running this function will update a tibble in
+your `gfile` object with publication information. You can also check your publication status by running `drive_check_publish()`.
+
```

Comment:
Do you think more people will connect with the word "revision" or "version"? I know "revision" is more technically correct, but suspect "version" resonates with more people. I would remove the sentence about updating a tibble in the `gfile` object, since the typical README reader won't want to know the internal stuff. I like the ability to check the publishing status! 💪